firebase-vscode/webviews/EmulatorPanel.tsx

```diff
@@ -33,14 +33,20 @@ const DEFAULT_EMULATOR_UI_SELECTIONS: EmulatorUiSelections = {
  */
 export function EmulatorPanel({
   firebaseJson,
+  projectId,
 }: {
   firebaseJson: FirebaseConfig;
+  projectId?: string | undefined;
 }) {
   if (!firebaseJson) {
     throw Error("Expected a valid FirebaseConfig.");
   }
+  var defaultState = DEFAULT_EMULATOR_UI_SELECTIONS;
```

Comment:
`let`, or actually probably use `const` and a ternary?

Reply:
Done